### DISCUSSION TITLE
Do i still need authme if im converting to online mode?

### DISCUSSION THREAD

**Discussion by NightLife-SG** [games647/FastLogin#735] [Q&A] [answered]
So i have been thinking of getting server back to online mode
i notice there a section in fast login that allow cracked player that has joined before able to join even when server in online mode
here a few question of me worrying

1.do i still need authme to run?
2.do i need to set onlinemode=true in server.properties?
3.will crack player lost they inventory?since it online mode,uuid must be different？if so ,is there a way i could save their progress/inventory?
4.What will happen to the uuid of the premium account?will it stay the same?

**Comment by TuxCoding** [comment, marked as answer]
>1.do i still need authme to run?
2.do i need to set onlinemode=true in server.properties?

https://github.com/games647/FastLogin#how-to-install

>3.will crack player lost they inventory?since it online mode,uuid must be different？if so ,is there a way i could save their progress/inventory?

Check the configuration file:

https://github.com/games647/FastLogin/blob/033fdd55ae105dd5bd0fb1709a87b29ee16e2981/core/src/main/resources/config.yml#L65

>4.What will happen to the uuid of the premium account?will it stay the same?

Yes depending on your config.

**Comment by NightLife-SG** [comment]
i dont understand what you mean by that link.
im be converting to online mode =true
so i still  be needing authme?
and do i need to set true on online mode = true in server.properties?
because in fastlogin there config to get this

**Reply by TuxCoding** [reply]
>so i still be needing authme?

An auth plugin is always required. FastLogin doesn't include the authentication functionality.

>and do i need to set true on online mode = true in server.properties?

Please look at the install guide on the project page. Offline mode is required. FastLogin will go into selective online mode if required.

**Comment by NightLife-SG** [comment]
but then what is this option for?kinda confuse me now
![online](https://user-images.githubusercontent.com/84533050/155289475-4cfaabb6-2451-4c27-96f5-903a1c654848.png)
if it say switch from offline mode to online mode

**Reply by TuxCoding** [reply]
You should change the server settings, so that the plugin can take this decision.

**Reply by NightLife-SG** [reply]
server setting such as?

**Reply by TuxCoding** [reply]
https://github.com/games647/FastLogin#spigotpaper-1

Turning off onlinemode.
